### ISSUE TITLE
Create good documentation on Access/Padme/RODBC problems & fix encountered with script_dataGrabFullLatLonOrGazLatLon_Socotra.R

### ISSUE THREAD

**Issue by FlicAnderson** [FlicAnderson/Scriptbox#12]
**TL;DR - when system resources exceeded, create temp table, run query in Padme/Accesss to put records into newly created temporary table, then pull into R & proceed as normal.**

Created table [FieldRexTemp] in test copy of Padme Arabia.

Output of sqlColumns() [here](https://gist.github.com/FlicAnderson/ad44350a62eb017387b6), shows column and data types

Ran [this query](https://gist.github.com/FlicAnderson/0a3ab3622c6902733f5b) to fill the FieldRexTemp table with the query result.  Had to do separate steps for create and "INSERT INTO FieldRexTemp SELECT ...<qry2>...;" because otherwise Access gave "system resource exceeded" error.

Running this worked after that:
`source("O://CMEP\ Projects/Scriptbox/database_connections/function_TESTPadmeArabiaCon.R")
TESTPadmeArabiaCon()
qry0 <- "SELECT * FROM FieldRexTemp"
fielRex <- sqlQuery(con_TESTPadmeArabia, qry0)`

All worked ok, can proceed as normal now.

I want to detail this better within the script for future issues like this.
